pkg/analytics/client.go

```diff
@@ -34,6 +34,7 @@ var (
 )
 
 var datadogLogLevel = "_logLevel"
+var datadogStatus = "status"
```

Comment:
So how does `status` differ from `_logLevel`? From the docs, it seems like `status is the correct one to use so why send both?

Reply:
yeh I was confused by this in their docs initially also. So _loglevel allows you to specify whatever level you want and you can filter on it from the DD console. The status one though only allows the pre-defined (info, warn, error) and we can't modify that, but that causes the colorized fields in DD 
![image](https://user-images.githubusercontent.com/92469267/212972225-f4c646f3-202d-4458-bfb0-9e7c4b6f7ffb.png)

So it's not entirely necessary since we filter on the _loglevel, but it gives you a quicker at a glance view since it will color the actual log entries instead of treating everything as info and blue by default.

Reply:
Can we switch our usage of `_loglevel` to `status`? I don't think we really care about the levels it doesn't support.

Reply:
we could, but it doesn't have things like debug/panic which currently i set to the closest relevant so debug -> info and panic -> error. If we're ok with losing granularity I can change that to just use status, but it didn't seem like we needed to consolidate and afaik it doesnt change costs in any way either so i left both.